pkg/cli/cmds/agent.go

```diff
@@ -151,7 +152,12 @@ var (
 		Usage:       "(agent/networking) Override default flannel cni config file",
 		Destination: &AgentConfig.FlannelCniConfFile,
 	}
-	ResolvConfFlag = &cli.StringFlag{
+	VPNAuthInfo = cli.StringFlag{
```

Comment:
I think this needs to 
* Be handled by the code that redacts secrets from the node-args annotation
* Have an environment variable so that it can be set without a CLI flag
* Maybe support reading from a file? I can already see the issues asking for this...

Reply:
Regarding the third point, are you suggesting to create a new cli flag which points at a file that contains the vpn-auth info?

Reply:
Yes, similar to what we do for the token (--token-file)